src/emqtt.erl

```diff
@@ -1195,6 +1318,12 @@ deliver(#mqtt_msg{qos = QoS, dup = Dup, retain = Retain, packet_id = PacketId,
     ok = eval_msg_handler(State, publish, Msg),
     State.
 
+now_ts() ->
+    erlang:system_time(millisecond).
+
+timestamp_to_ms(Ts) ->
```

Comment:
why not upgrade Ts from tuple to integer

Reply:
Mainly to change the timestamp for the flight window from `os:timesamp/0` to `erlang:system_time/1`